### ISSUE TITLE
loop "docker-entrypoint.sh: Waiting for control socket to be removed..."

### ISSUE THREAD

**Issue by nergalex** [nginx/unit#677]
DESCRIPTION
Container logs show an infinite loop of record `/usr/local/bin/docker-entrypoint.sh: Waiting for control socket to be removed...`
Impact: Unit is not responding
```
# curl --unix-socket /var/run/control.unit.sock http://localhost/config/ -vvv
*   Trying /var/run/control.unit.sock:0...
* Connected to localhost (/var/run/control.unit.sock.tmp) port 80 (#0)
> GET /config/ HTTP/1.1
> Host: localhost
> User-Agent: curl/7.74.0
> Accept: */*
> 
^C
```

CONTEXT
Unit version: 1.26.1
Docker logs: 
[unit-debug.log.txt](https://github.com/nginx/unit/files/8484761/unit-debug.log.txt)
Dockerfile: [here](https://github.com/nergalex/f5-xc-logstream/blob/master/build-ce/Dockerfile)
Unit version: 1.26.1
```
# unitd --version
unit version: 1.26.1
configured as ./configure --prefix=/usr --state=/var/lib/unit --control=unix:/var/run/control.unit.sock --pid=/var/run/unit.pid --log=/var/log/unit.log --tmp=/var/tmp --user=unit --group=unit --openssl --libdir=/usr/lib/x86_64-linux-gnu --cc-opt='-g -O2 -ffile-prefix-map=/unit=. -specs=/usr/share/dpkg/no-pie-compile.specs -fstack-protector-strong -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fPIC' --ld-opt='-specs=/usr/share/dpkg/no-pie-link.specs -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -pie' --modules=/usr/lib/unit/modules
```

**Comment by nergalex** [comment]
Workaround (dirty): force kill unit in docker-entrypoint.sh
Example [here](https://github.com/nergalex/f5-xc-logstream/blob/master/build-ce-workaround/logstream-xc.sh)
Logs:
[unit-ok.log](https://github.com/nginx/unit/files/8484941/unit-ok.log)

**Comment by tippexs** [comment]
We are currently working on a fix for #570 . Are you able to share the unit configuration? Do you have any sort of outgoing database connections or similar things in your application?

**Comment by nergalex** [comment]
Hi Tippexs,

Correct, good catch, a Thread was not set as [daemon ](https://docs.python.org/3/library/threading.html#threading.Thread.daemon) in App code. Now it's OK.

Thank you !
